.gitignore

```diff
@@ -61,3 +61,6 @@ tests/test-team/test-results/
 tests/test-team/playwright-report/
 tests/test-team/blob-report/
 tests/test-team/playwright/.cache/
+
+#plugin-cache
```

Comment:
```suggestion

```

Reply:
I have done that now

Reply:
oh actually, doesn't let me push: 
`remote: error: See https://gh.io/lfs for more information.
remote: error: File plugin-cache/registry.terraform.io/hashicorp/aws/5.64.0/darwin_amd64/terraform-provider-aws_v5.64.0_x5 is 551.40 MB; this exceeds GitHub's file size limit of 100.00 MB`

Reply:
sorry my bad, done